### PR TITLE
[BZ 1662727] Include variable "openshift_cloudprovider_openstack_conf_file" for OSP installation 

### PR DESCRIPTION
**Pull request by maxwelldb**
Context: https://bugzilla.redhat.com/show_bug.cgi?id=1662727

### CODE REVIEW

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL. Tiny change!

File: install_config/topics/ocp_osp_provisioning.adoc [hunk 1]
```diff
@@ -302,6 +302,7 @@ openshift_cloudprovider_openstack_password: "{{ lookup('env','OS_PASSWORD') }}"
 openshift_cloudprovider_openstack_tenant_name: "{{ lookup('env','OS_PROJECT_NAME') }}"
 openshift_cloudprovider_openstack_blockstorage_version: v2
 openshift_cloudprovider_openstack_domain_name: "{{ lookup('env','OS_USER_DOMAIN_NAME') }}"
+openshift_cloudprovider_openstack_conf_file: PATH_TO_LOCAL_OPENSTACK_CONFIGURATION_FILE
```
Comment (bergerhoffer) [inline install_config/topics/ocp_osp_provisioning.adoc:305]: Is `PATH_TO_LOCAL_OPENSTACK_CONFIGURATION_FILE` an actual env variable, or is it something the user needs to replace? If the user needs to replace, then shouldn't it be lowecase and in angle brackets?

**Review by bergerhoffer — COMMENTED** [review]
A question for ya

File: install_config/topics/ocp_osp_provisioning.adoc (hunk 1 above)
Comment (maxwelldb) [reply install_config/topics/ocp_osp_provisioning.adoc:305]: Got the proposed new style in my head. Subbing.

**Pull request merged**